test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssert.cs

```diff
@@ -20,30 +22,62 @@ public static class XmlAssert
         /// <param name="actualXml">Actual xml string.</param>
         public static void Equal(string expectedXml, string actualXml)
         {
-            var sortedExpectedXDoc = SortAttributes(XDocument.Parse(expectedXml));
```

Comment:
Since 'expectedXml` could be handcrafted when writing a test, it would be better to know that the xml is invalid.